frontend/packages/dev-console/src/components/topology/components/nodes/PodSet.tsx

```diff
@@ -77,7 +77,7 @@ const PodSet: React.FC<PodSetProps> = ({ size, data, x = 0, y = 0, showPodCount
   );
   const hpaControlledScaling = !!hpa;
 
-  const obj = get(data, ['current', 'obj'], null) || data.dc;
+  const obj = RevisionModel.kind === data.dc.kind ? data.dc : data.current.obj || data.dc;
```

Comment:
```suggestion
  const obj = RevisionModel.kind === data.dc.kind ? data.dc : data?.current?.obj || data.dc;
```

Reply:
Thanks Jeff, Data will always be there it's required prop or we see scenario where it can be undefined?